resources/qml/Root.qml

```diff
@@ -116,6 +129,39 @@ Page {
         }
     }
 
+    Connections {
+        target: Rooms.currentRoom
+        onOpenRoomMembersDialog: {
+            var membersDialog = roomMembersComponent.createObject(timelineRoot, {
+                "members": members,
+                "roomName": Rooms.currentRoom.roomName
+            });
+            membersDialog.show();
+        }
+    }
+
+    Connections {
+        target: Rooms.currentRoom
+        onOpenRoomSettingsDialog: {
+            var roomSettings = roomSettingsComponent.createObject(timelineRoot, {
+                "roomSettings": settings
+            });
+            roomSettings.show();
+        }
+    }
+
+    Connections {
+        target: Rooms.currentRoom
+        onOpenInviteUsersDialog: {
+            var dialog = inviteDialog.createObject(timelineRoot, {
+                "roomId": Rooms.currentRoom.roomId,
+                "roomName": Rooms.currentRoom.roomName,
+                "invitees": invitees
+            });
+            dialog.show();
+        }
+    }
```

Comment:
You can put all of these in once Connections object.

Reply:
Are lines 143-151 even necessary or did I manage to duplicate something?

Reply:
Hmm, looks like they are duplicated in TimelineView.qml. I'll remove it from TimelineView.qml since that makes the most sense